lib/models/movie.js

```diff
@@ -4,6 +4,29 @@ const Bookshelf = require('../libraries/bookshelf');
 
 module.exports = Bookshelf.Model.extend({
   tableName: 'movies',
+
+  filterByYear (release_year, release_year_start, release_year_end) {
+    return this.query((qb) => {
+      if (release_year) {
+        qb.where('release_year', '=', release_year);
+      }
+      if (release_year_start) {
+        qb.where('release_year', '>=', release_year_start);
+      }
+      if (release_year_end) {
+        qb.where('release_year', '<=', release_year_end);
+      }
+    });
+  },
+
+  filterByTitle (title) {
+    return this.query((qb) => {
+      if (title) {
+        qb.where('name', 'LIKE', `%${title}%`);
+      }
+    });
+  },
```

Comment:
Functionality looks good! Here are some style points that are optional.

1. We tend not to use the ES6 method shorthand for objects (we only really do it for `class`es so far), but we do use the property shorthand. [Relevant but somewhat out of date wiki document](https://github.com/lob/wiki/blob/master/engineering/style-guides/es6.md). Sorry I missed this when you wrote `serialize`.

2. Typically we use camelCase for variables internally, but expose snake_case in keys for responses or in input validation. So basically the HTTP query params you've got are properly named, but the parameter names to these functions should be more like `releaseYear`.

3. Typically when we write filters for models, we do it as a single method. This way the controller only ever calls `filter` once while the `filter` object passed in can contain a large variety of inputs that the `filter` function goes through. We have a [few](https://github.com/lob/hapi-query-filter) plugins [that are standardized](https://github.com/lob/hapi-bookshelf-total-count) around this convention. I recommend taking a look at the `Postcard` model in the API to get a sense of how that looks in practice.